apps/prairielearn/src/lib/assessment.js

```diff
@@ -36,486 +38,414 @@ const InstanceLogSchema = z.object({
 });
 
 /**
- * Assessment module.
- * @module assessment
+ * Check that an assessment_instance_id really belongs to the given assessment_id
+ *
+ * @param {string} assessment_instance_id - The assessment instance to check.
+ * @param {string} assessment_id - The assessment it should belong to.
  */
+export async function checkBelongsAsync(assessment_instance_id, assessment_id) {
+  if (
+    (await sqldb.queryRow(
```

Comment:
`queryRow` will throw its own error if a row is not found. I believe you want `queryOptionalRow` here instead.

Reply:
Fixed in 1daba6c17c71e2bde978d382c2551ddcbe09a027.